examples/scenarioSunlineFiltersCompare.py

```diff
@@ -0,0 +1,352 @@
+        CSS.stateInMsg.subscribeTo(scObject.scStateOutMsg)
+        CSS.this.disown()
+    for CSSHat in CSSOrientationList:
+        newCSS = coarseSunSensor.CoarseSunSensor()
+        newCSS.modelTag = "CSS" + str(counter)
+        counter += 1
+        setupCSS(newCSS)
+        newCSS.nHat_B = CSSHat
+        cssConstelation.appendCSS(newCSS)
+    scSim.AddModelToTask(simTaskName, cssConstelation)
+
+    #
+    #   add the FSW CSS information
+    #
+    cssConstVehicle = messaging.CSSConfigMsgPayload()
+
+    totalCSSList = []
+    for CSSHat in CSSOrientationList:
+        newCSS = messaging.CSSUnitConfigMsgPayload()
+        newCSS.nHat_B = CSSHat
+        newCSS.CBias = 1.0
+        totalCSSList.append(newCSS)
+    cssConstVehicle.nCSS = len(CSSOrientationList)
+    cssConstVehicle.cssVals = totalCSSList
+
+    cssConstMsg = messaging.CSSConfigMsg().write(cssConstVehicle)
+
+    #
+    # Setup filter
+    #
+    numStates = 6
+    bVecLogger = None
+
+    if FilterType == 'uKF':
+        module = sunlineUKF.SunlineUKF()
+        module.modelTag = "SunlineUKF"
+        setup_ukf_data(module)
+        # Add test module to runtime call list
+        scSim.AddModelToTask(simTaskName, module)
+        simpleNavMsg = messaging.NavAttMsg()
+        navLog = module.navStateOutMsg.recorder()
+        filtLog = module.filtDataOutMsg.recorder()
+        css_measurement_noise_std = np.sqrt(module.qObsVal)
+
+    if FilterType == 'SRuKF':
+        module = sunlineSRuKF.SunlineSRuKF()
+        module.modelTag = "SunlineSRuKF"
+        setup_srukf_data(module)
+        # Add test module to runtime call list
+        scSim.AddModelToTask(simTaskName, module)
+        simpleNavMsg = messaging.NavAttMsg()
+        module.navAttInMsg.subscribeTo(simpleNavMsg)
+        navLog = module.navAttOutMsg.recorder()
+        filtLog = module.filterOutMsg.recorder()
+        css_residual_data_log = module.filterCssResOutMsg.recorder()
+        scSim.AddModelToTask(simTaskName, css_residual_data_log)
+        css_measurement_noise_std = module.getCssMeasurementNoiseStd()
+
+    module.cssDataInMsg.subscribeTo(cssConstelation.constellationOutMsg)
+    module.cssConfigInMsg.subscribeTo(cssConstMsg)
+
+    scSim.AddModelToTask(simTaskName, navLog)
+    scSim.AddModelToTask(simTaskName, filtLog)
+
+    #
+    #   initialize Simulation
+    #
+    scSim.InitializeSimulation()
+
+    #
+    #   configure a simulation stop time and execute the simulation run
+    #
+    scSim.ConfigureStopTime(simulationTime)
+
+    # Time the runs for performance comparisons
+    start_time = time.time()
+    scSim.ExecuteSimulation()
+    end_time = time.time() - start_time
+
+    #
+    #   retrieve the logged data
+    #
+    def addTimeColumn(time, data):
+        return np.transpose(np.vstack([[time], np.transpose(data)]))
+
+    # Get messages that will make true data
+    timeAxis = dataLog.times()
+    OutSunPos = addTimeColumn(timeAxis, sunLog.PositionVector)
```

Comment:
:small_red_triangle_down: change all these variables to be lower camel case e.g. `camelCase`

Also is this `addTimeColumn` call needed? Is there not a `sunLog.times()`?